Sources/ZcashLightClientKit/Tool/DerivationTool.swift

```diff
@@ -108,8 +107,11 @@ public class DerivationTool: KeyDeriving {
     /// - Parameter numberOfAccounts: the number of accounts to use. Multiple accounts are not fully
     /// supported so the default value of 1 is recommended.
```

Comment:
Please check that this is correct.
```suggestion
    /// - Parameter accountIndex: the ZIP 32 index of the account
```

Reply:
Updated